packages/functional-tests/tests/settings/changeEmail.spec.ts

```diff
@@ -140,7 +140,9 @@ test.describe('severity-1 #smoke', () => {
       await signin.fillOutEmailFirstForm(initialEmail);
       await signin.fillOutPasswordForm(newPassword);
 
-      await expect(settings.primaryEmail.status).toHaveText(credentials.email);
+      await expect(settings.primaryEmail.status).toHaveText(credentials.email, {
```

Comment:
What would you think about waiting for the settings page to be rendered instead of adding a timeout? Wondering if this would resolve the issue here.

E.g.,

``` await expect(settings.settingsHeading).toBeVisible();```